### PR TITLE
 fix cluster_status_matches_regex to match case insensitive strings

### PR DESCRIPTION
**Pull request by rjagu1**
This PR is about the match case insensitive string in the cluster_status_matches_regex function (lib/hacluster.pm)

- Related ticket: [TEAM-8594](https://jira.suse.com/browse/TEAM-8594)
- Verification run: Azure @ http://openqaworker15.qa.suse.cz/tests/244007#step/test_cluster/456
                                            http://openqaworker15.qa.suse.cz/tests/244006#step/test_cluster/456
                               AWS :http://openqaworker15.qa.suse.cz/tests/244009#
                               GCP :http://openqaworker15.qa.suse.cz/tests/244008# 
Latest VR : http://openqaworker15.qa.suse.cz/tests/245027#step/test_cluster/456
                    http://openqaworker15.qa.suse.cz/tests/245028#step/test_cluster/456
                    http://openqaworker15.qa.suse.cz/tests/245029

### CODE REVIEW

**Review by mpagot — APPROVED** [review]
LGTM

File: lib/hacluster.pm [hunk 1]
```diff
@@ -1219,7 +1219,7 @@ sub cluster_status_matches_regex {
     my $previous_line = '';
 
     for my $line (split("\n", $show_cluster_status)) {
-        if ($line =~ /(Stopped:| Stopped|Failed:| Failed|Pending:| Pending|Blocked:| Blocked)/) {
+        if (lc($line) =~ /\s?(stopped|failed|pending|blocked|starting|promoting):?/) {
```
Comment (alvarocarvajald) [inline lib/hacluster.pm:1222]: Suggestion:
```
if ($line =~ /\s?(stopped|failed|pending|blocked|starting|promoting):?/i) {
```
`i` means insensitive :)

**Review by alvarocarvajald — APPROVED** [review]
A small suggestion to make the code match better the title of the PR, but it LGTM anyways.

File: lib/hacluster.pm (hunk 1 above)
Comment (lpalovsky) [reply lib/hacluster.pm:1222]: Oh.. this is what we were searching for yesterday! :D

**Review by lpalovsky — APPROVED** [review]
LGTM

File: lib/hacluster.pm (hunk 1 above)
Comment (mpagot) [reply lib/hacluster.pm:1222]: https://regex101.com/r/FIrl0n/1 and the ticket has some comment with explanation about it

**Review by mpagot — APPROVED** [review]
LGTM

**Pull request merged**